build-system/pr-check/build-targets.js

```diff
@@ -148,6 +149,17 @@ const targetMatchers = {
       file.startsWith('build-system/babel-config/')
     );
   },
+  [Targets.BUILD_SYSTEM]: (file) => {
+    if (isOwnersFile(file)) {
+      return false;
+    }
+
+    return (
+      file == 'build-system/tsconfig.json' ||
+      (file.startsWith('build-system') &&
+        (file.endsWith('.js') || file.endsWith('.ts')))
```

Comment:
Since there are several json files in build-system that are consumed by js files, it might make sense to also include them in this list.

```suggestion
        (file.endsWith('.js') ||
          file.endsWith('.ts') ||
          file.endsWith('.json')))
```

Reply:
That makes sense. With the new file extension we don't need the explicit check for `tsconfig.json` but would you like to leave it there for the reasons stated in [you other comment](https://github.com/ampproject/amphtml/pull/34544#discussion_r639986254)?

Reply:
I like the idea of leaving it in there as an explicit condition, as you've already done :)